CMakeLists.txt

```diff
@@ -40,6 +40,11 @@
 # Then you can just do "make"; cmake will automatically be re-run by make since the CMakeLists.txt
 # file has changed.
 
+# By default, this will make a serial execution build but can be changed by turning on the parallel flag
+#
+#   mkdir Debug
+#   cd Debug
+#   Cmake -D Parallel=ON
```

Comment:
And since, IIRC, the default build is a Release build, if you're going to use a folder called "Debug" here then you ought to supply the `-D CMAKE_BUILD_TYPE=Debug` flag.  But probably you ought to show how to make a parallel Release build, not a Debug build, anyway; presumably that will be what most people want.

Reply:
It should work with both, but I changed it nonetheless